packages/testing-library/utils/test.ts

```diff
@@ -13,8 +13,9 @@ export async function runTests(page: Page, adder: AdderWithoutExplicitArgs, opti
         click: async (selector, waitForNavigation) => {
             await click(page, selector, waitForNavigation);
         },
+        // eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
why remove the `await` if it causes you to add this?

Reply:
Because `expectUrlPath` is non async function, and the await was misplaced previously. 

If I also remove the `async` keyword from this `expectUrlPath: async (path) => {` the types are also not happy because the tests should be an async function. Is there any other way to solve this?